src/app/components/alert/alert.component.ts

```diff
@@ -0,0 +1,56 @@
+import { CommonModule } from "@angular/common";
+import { Component, HostBinding, Input, OnInit, ViewEncapsulation } from "@angular/core";
+import { State } from "src/app/enums/state";
+import { Button } from "src/app/interfaces/button";
+import { v4 as uuid } from "uuid";
+
+@Component({
+  selector: "adv-alert",
+  standalone: true,
+  templateUrl: "./alert.component.html",
+  styleUrl: "./alert.component.scss",
+  encapsulation: ViewEncapsulation.None,
+  imports: [CommonModule]
+})
+export class AlertComponent implements OnInit {
+  public id!: string;
+  public State = State;
+  public visible: boolean = false;
+
+  @Input()
+  public title: string | undefined;
```

Comment:
It depends; if variables can be instantiated with a default value, then it is better to avoid `undefined`. However, if that's not the case using `undefined` is better in my opinion, because it meaningfully conveys in code that a variable was never instantiated and might be discarded for that reason.